samplestore/src/main/java/com/stripe/samplestore/StoreCart.java

```diff
@@ -0,0 +1,101 @@
+package com.stripe.samplestore;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+
+import java.util.ArrayList;
+import java.util.Currency;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.UUID;
+
+public class StoreCart implements Parcelable {
```

Comment:
A simple version of a Cart, without all the various items that Google's android pay carts have (those are no longer used with Pay with Google). Users will be tasked with creating their own carts now, so this is just an example of an easy one.